src/core/aws-client.ads

```diff
@@ -70,7 +70,7 @@ package AWS.Client is
    --  Number of time a data is requested from the Server if the first
    --  time fails.
 
-   HTTP_Default : HTTP_Protocol renames HTTPv1;
+   HTTP_Default : HTTP_Protocol renames HTTPv2;
```

Comment:
I guess you just forgot to keep HTTPv1 default.

Reply:
Indeed ! Now fixed.